### DISCUSSION TITLE
Does edge support redis sentinel mode ?

### DISCUSSION THREAD

**Discussion by LittleHawk03** [Unleash/unleash#6509] [Q&A] [answered]
Does the Unleash Edge support connect redis + sentinel for HA purposes?

**Comment by chriswk** [comment]
So, I'm not sure what you are asking. Edge does not run Redis, it connects to Redis. Does connection properties change for a redis client if it is connected to a sentinel cluster instead of a normal redis server?

**Reply by sighphyre** [reply]
Hey @LittleHawk03, currently no, there's no support for sentinel at the moment. If this is important to you, would you mind opening a ticket on the Unleash Edge repo?

**Reply by LittleHawk03** [reply]
Let me ask, edge uses redis for the sole purpose of persistant storage and does it help increase edge performance?

**Reply by sighphyre** [reply, marked as answer]
So sorry @LittleHawk03, completely missed your response here.

Redis will only improve the startup time of Edge. Edge will favour the cache in Redis over going directly to the upstream Unleash instance to sync features and auth tokens. So you may see a small improvement between the start and the ready state. The biggest advantage you get from Redis here is that if there's no network connectivity to your Unleash instance, Edge will still serve requests from it's previous cache

Once Edge is up and running, it'll serve all it's requests directly from it's own in process memory, Redis doesn't play a part in that so no performance change at steady state
